app/assets/javascripts/groups.coffee

```diff
@@ -1,4 +1,65 @@
+select2GroupMemberFormatter = (result) ->
+  result.text + ' <small><b>' + result.id + '</b></small>'
+
+select2Sorter = (a, b) ->
+  if (a > b)
+    return 1
+  if (a < b)
+    return -1
+  return 0
+
+select2HTMLSorter = (a, b) ->
+  select2Sorter(a.title, b.title)
+
+select2JSONSorter = (a, b) ->
+  select2Sorter(a.text, b.text)
+
+setCurrentGroupMembers = ->
+  $.each $('.urs_autocomplete'), ->
+    selectedUsers = $.map $(this).find('option'), (option) -> 
+      $(option).val()
+
+    $(this).val(selectedUsers).trigger('change')
+
 $(document).ready ->
+  $('.urs_autocomplete').select2
```

Comment:
Since most of the group members JavaScript is now in this file, `groups_dual_select.coffee` is not being used and can be removed now, right?